src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class WorkWithFile {
+    private static final String COMMA_SPLITTER = ",";
+    private static final int OPERATION_NAME_INDEX = 0;
+    private static final int OPERATION_AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFromFile(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        int supply = getOperationAmount(readFromFile(fromFileName), "supply");
+        int buy = getOperationAmount(readFromFile(fromFileName), "buy");
+        stringBuilder.append("supply")
+                .append(",")
+                .append(supply)
+                .append(System.lineSeparator())
+                .append("buy")
+                .append(",")
+                .append(buy)
+                .append(System.lineSeparator())
+                .append("result,")
+                .append(supply - buy);
+        writeToFile(stringBuilder.toString(), toFileName);
```

Comment:
Refactor getStatistic this way:
```suggestion
        String[] dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```